main/solution/backend/config/settings/.defaults.yml

```diff
@@ -1,5 +1,6 @@
 # Options used when issuing JWT token such as which algorithm to use for hashing and how long to keep the tokens alive etc
-jwtOptions: '{"algorithm":"HS256","expiresIn":"2 days"}'
+# Note that default expiry of access token is set to one hour. This will force the user to logout and login back after an hour
+jwtOptions: '{"algorithm":"HS256","expiresIn":"1 hours"}'
```

Comment:
Could you explain how you tested this? and was it successful?

Reply:
talked offline; tested and works as expected

Reply:
Did couple of tests:
1. Unpacked the token in jwt.io and verified that the token had the correct `exp` set
2. Used the token after an hour and verified that API calls were unsuccessful and I got an `unauthorized` error